src/coord/src/coord.rs

```diff
@@ -3303,7 +3365,13 @@ impl Coordinator {
             id,
             values: values.lower(),
         };
-        self.sequence_insert(session, plan)
+
+        let constants = self
+            .prep_relation_expr(plan.values, ExprPrepStyle::Write)?
```

Comment:
Can probably just avoid making the `plan` variable here since it's not really used.

Reply:
Good call